packages/flutter/lib/src/material/app.dart

```diff
@@ -57,6 +57,11 @@ enum ThemeMode {
 /// adding material-design specific functionality, such as [AnimatedTheme] and
 /// [GridPaper].
 ///
+/// The [MaterialApp] passes a fallback [TextStyle] to [WidgetsApp] which is used in
+/// absense of [Material] widgets. The [Material] widgets builds a [DefaultTextStyle]
```

Comment:
absense => absence

of [Material] widgets => of a [Material] widget

Reply:
Thanks for the review. 
I have updated the documentation and added troubleshooting with screenshot and a fix 

> you could do a little better by using a screenshot with some printable text.

By screenshot with some printable text,  do you mean the type of screenshot with text description?
Please explain, thanks.
![text_without_material](https://user-images.githubusercontent.com/48603081/149316729-f79e60b3-6a21-4c64-9b46-5750db5e70eb.png)

Reply:
This screenshot looks good, although it would be OK to leave out the "Text widget doesn't..." label and arrow.

Also: https://github.com/flutter/flutter/pull/96530#discussion_r784197229